Gemfile

```diff
@@ -63,6 +63,7 @@ group :development do
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'travis'
+  gem 'irb'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `irb` should appear before `travis`.](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5446585)